new/ohai-new-plugins-policy.md

```diff
@@ -0,0 +1,45 @@
+---
+RFC: unassigned
+Title: Ohai new plugins policy
+Author: Phil Dibowitz <phil@ipom.com>
+Status: Draft
+Type: <Standards Track, Informational, Process>
+---
+
+# Ohai new plugins policy
+
+To allow Chef to move fast while still providing stability, this lines out a
+policy for clarity.
+
+## Motivation
+
+    As a Chef developer,
+    I want to be able to provide new functionality to the community
+    so that they can use it.
+
+    As a Chef user,
+    I want to be able to leverage new functionality as quickly as possible
```

Comment:
I don't think this reflects anything but the smallest minority of our user base at this point.